package.json

```diff
@@ -43,6 +43,6 @@
     "date-fns": "^2.28.0"
   },
   "engines": {
-    "npm": "~8.19"
+    "npm": ">=9.5.1 <10"
```

Comment:
@JSv4 Node 18.16 ships with `npm` 9.5.1 so without this there were warnings: 

```
pj@LTSE-Q293FVD2T0 Open-Cap-Format-OCF % npm install
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: undefined,
npm WARN EBADENGINE   required: { npm: '~8.19' },
npm WARN EBADENGINE   current: { node: 'v18.16.0', npm: '9.5.1' }
npm WARN EBADENGINE }
```

so I updated this.